drivers/storage/libstorage/libstorage_client_api.go

```diff
@@ -255,6 +261,16 @@ func (c *client) VolumeAttach(
 	request *types.VolumeAttachRequest) (*types.Volume, string, error) {
 
 	ctx = c.withInstanceID(c.requireCtx(ctx), service)
+	ctxA, err := c.withAllLocalDevices(ctx)
```

Comment:
Actually, AllInstanceIDs doesn't make sense. I'll just remove one of the AllLocalDevices.